nibabel/freesurfer/io.py

```diff
@@ -73,6 +73,26 @@ def _read_volume_info(fobj):
     return volume_info
 
 
+def _pack_rgba(rgba):
+    """Used by :meth:`read_annot` and :meth:`write_annot` to pack an RGBA
```

Comment:
I know this is a private function, but the first line should be standalone with further explanation after a blank line.  How about

```
""" Pack an RGBA sequence into a single integer

Used by :meth:`read_annot` and :meth:`write_annot`.
```